External Link Helper

nakalsio opened the pull request:
## I am submitting a new Community Plugin

## Repo URL
Link to my plugin: [https://github.com/nakalsio/obsidian-danpung](https://github.com/nakalsio/obsidian-danpung)

## Release Checklist

- [x] I have tested the plugin on
  - [x] Windows
  - [x] macOS
  - [x] Linux
- [x] My GitHub release contains all required files
  - [x] main.js
  - [x] manifest.json
  - [x] styles.css
- [x] GitHub release name matches the exact version number specified in my manifest.json (Note: Use the exact version number, don't include a prefix v
- [x] The id in my manifest.json matches the id in the community-plugins.json file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
- [x] I have given proper attribution to these other projects in my README.md.

nakalsio commented:
Changed the tag from 'v0.8.0' to '0.8.0'

liamcain commented:
- [ribbonIconEl.addClass('my-plugin-ribbon-class');](https://github.com/nakalsio/obsidian-danpung/blob/5b881ed49e1ed223fcb32034d13d2e56f4eb43c8/src/main.ts#L37) you can remove this line.
- [modifiers: ["Shift", "Alt"], key: ";"](https://github.com/nakalsio/obsidian-danpung/blob/5b881ed49e1ed223fcb32034d13d2e56f4eb43c8/src/main.ts#L47) While there's no hard rule against it, we recommend against providing a default hotkey when possible. The reason being that it's easy to pick a hotkey that a user already has configured and they could get confused when that key doesn't do what they expect. Also, it's hard choosing a safe default hotkey that's available for all operating systems. For example, Shift+Alt hotkeys won't work for macOS since that is reserved for inputting unicode characters.
- [getCache(file.path)](https://github.com/nakalsio/obsidian-danpung/blob/5b881ed49e1ed223fcb32034d13d2e56f4eb43c8/src/main.ts#L96) assign this to a variable so you don't need to call getCache multiple times in a row.

nakalsio commented:
1. Removed `ribbonIconEl.addClass('my-plugin-ribbon-class');`
2. Removed hotkeys
3. Assigned to `theCache` : https://github.com/nakalsio/obsidian-danpung/blob/8b929e5aadfb1dbfd7b970f7602f40f9dfca1708/src/main.ts#L92
4. gifs are looping
